### ISSUE TITLE
Toggle switch lacks focused outline

### ISSUE THREAD

**Issue by salle18** [axa-ch/patterns-library#2079]
@axa-ch/toggle-switch v2.0.1

When using keyboard to navigate through form elements toggle switch doesn't have any visual indication that it is being focused.

Further reading:
https://www.a11yproject.com/posts/2013-01-25-never-remove-css-outlines/

**Comment by markus-walther** [comment]
Yes, this is an oversight. To be clarified with designers: The visible focus frame should be rectangular or rounded?

**Comment by marcwelz** [comment]
The frame should be rounded.
[https://www.figma.com/file/6zurYk3bJpzUg0H2THSxGF/AXA-UI-Kit?node-id=1765%3A888](url)

**Comment by marcwelz** [comment]
This issue is resolved
